CONTRIBUTING.md

```diff
@@ -108,6 +108,15 @@ Here are some example pull requests:
 
   - Adding a gutter icon: [#758](https://github.com/intellij-rust/intellij-rust/pull/758).
 
+And also a tutorial series "Contributing to Intellij-Rust" by @kobzol:
```

Comment:
```suggestion
And also a tutorial series "Contributing to Intellij-Rust" by [@Kobzol](https://github.com/Kobzol):
```
Otherwise, it won't be rendered as expected